src/classes/CSVIterable.cls

```diff
@@ -0,0 +1,62 @@
+/**
+* File Name :   CSVIterable.cls
+* Description : Class defining a new type of Iterable, similar to list but destructive after read
+*			   Modified version of http://developer.financialforce.com/customizations/importing-large-csv-files-via-batch-apex/
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 02/04/2014 Klea Kolaric   Initial version
+*
+*
+*/
+public with sharing class CSVIterable implements Iterator<string>, Iterable<string>
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.